src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/DashboardClaimInfoService.java

```diff
@@ -111,6 +111,7 @@ private DashboardClaimInfo translateCaseDataToDashboardClaimInfo(CaseDetails cas
             .defendantName(nonNull(caseData.getRespondent1()) ? caseData.getRespondent1().getPartyName() : null)
             .claimAmount(nonNull(caseData.getTotalClaimAmount()) ? caseData.getTotalClaimAmount() : null)
             .admittedAmount(caseData.getPartAdmitPaidValuePounds())
+            .responseDeadlineTime(caseData.getRespondent1ResponseDeadline())
```

Comment:
does it work for cmc claims? don't you need to convert to localdatetime from localdate response deadline for cmc?

Reply:
please have a look at ClaimStoreService line 53

Reply:
I added a new field not to impact other flows

Reply:
ok but dont we need to this calculation for cmc claims too?

Reply:
cmc added

Reply:
Resolving the conversation. @TatsyMaslova is advised to check the comment again when she returns.